### PR TITLE
Srl 702  webform translate button

### PR DESCRIPTION
**Pull request by reshmai**
Webform translate button added in header with translations links

### CODE REVIEW

File: web/themes/custom/atrium/includes/page.inc [hunk 1]
```diff
@@ -185,10 +186,30 @@ function _atrium_translation_menu() {
       ? str_replace("/$langcode", '', $current_path)
       : $current_path;
   }
-  // Create links for dropbutton component if translated content exists for page.
+  // Create links for dropbutton component if translated content or webform exists for page.
   $links = [];
   foreach ($languages as $langcode => $language) {
     $node = \Drupal::routeMatch()->getParameter('node');
+    $webform = \Drupal::routeMatch()->getParameter('webform');
```
Comment (ivanbueno-jcc) [inline web/themes/custom/atrium/includes/page.inc]: This 2 lines can be taken outside of the foreach loop.

File: web/themes/custom/atrium/includes/page.inc [hunk 2]
```diff
@@ -185,10 +186,30 @@ function _atrium_translation_menu() {
...
+      $langcode = $langcode == $default_language->getId() ? '' : "/$langcode";
+      if($webform != null
+        && $prefixed_translated_path != $prefixed_current_path
+        && $prefixed_translated_path != '/en' . $prefixed_current_path){
+        $links[] = [
+          'title' => $language->getName(),
+          'href' => "$host$langcode$translated_path",
+        ];
+
+      }
```
Comment (ivanbueno-jcc) [inline web/themes/custom/atrium/includes/page.inc]: This whole block is similar to the node block below.  You can collapse them, and pass in the variables for lines that are different between the two.

**Review by ivanbueno-jcc — CHANGES_REQUESTED** [review]

File: web/themes/custom/atrium/includes/page.inc (hunk 1 above)
Comment (reshmai) [reply web/themes/custom/atrium/includes/page.inc]: Moved lines out of the foreach

File: web/themes/custom/atrium/includes/page.inc (hunk 2 above)
Comment (reshmai) [reply web/themes/custom/atrium/includes/page.inc]: Created separate function for same set of code for webform and node.I had to change code little more to fulfill conditions for node and webform. Please let me know if I need to change anymore or any code refactor needs to be done. Thanks!

**Review by ivanbueno-jcc — APPROVED** [review]

**Pull request merged**